docs/integration-testing-journey.md

```
@@ -302,11 +308,117 @@ _SPO2_PREFIX = "vital_signs/vital_signs:0/indirect_oximetry"
+curl -u ehrbase-user:SuperSecretPassword \
+  -H "Accept: application/openehr.wt+json" \
+  "http://localhost:8080/ehrbase/rest/openehr/v1/definition/template/adl1.4/IDCR%20-%20Vital%20Signs%20Encounter.v1"
+```
+
+**Key Finding**: The `application/openehr.wt+json` media type was not available in EHRBase 2.0.0 but works in 2.26.0.
+
+### Web Template Analysis
+
+The web template shows a flattened structure where data elements are direct children of observations:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)